websites/W/Wallhaven/dist/metadata.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "Cascade",
+    "id": "442612628104151040"
+  },
+  "service": "Wallhaven",
+  "description": {
+    "en": "Your source for the best high quality wallpapers on the Net!"
+  },
+  "url": "wallhaven.cc",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/SsIZl80.png",
+  "thumbnail": "https://i.imgur.com/QV6vAk6.jpg",
+  "color": "#0E6288",
+  "iframe": false,
+  "tags": [
```

Comment:
no need to set it to false since thats the default value

Reply:
I've committed the requested changes 👌